PayloadPackaging/PayloadPackaging.csproj

```diff
@@ -74,6 +74,14 @@
       <Name>SonarRunner.Shim</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup Condition="$(Configuration)== 'Debug'">
```

Comment:
I changed the payload packaging project to include .pdb files when in DEBUG mode to help debugging. I suppose I could include them in the RELEASE version as well but it will make the zip file quite a bit larger so I won't do that.